### DISCUSSION TITLE
备注自己的问题（安装好后，you-get命令出现"bash: you-get: command not found"）&WIN10系统解决方法

### DISCUSSION THREAD

**Discussion by YDTRINLEY** [LussacZheng/video-downloader-deploy#21] [General]
**问题：**
按照说明下载安装了video-downloaader-deploy之后，在git bash输入命令 `you-get`，一直显示 `bash: you-get: command not found` 
**解决方法：**
把 ...\video-downloader-deploy\usr\python-embed\Scripts  的完整路径放到 用户变量PATH里面就好了 

后记（blabla……）：
本来是想下载安装you-get的，但是试了官方所有方法pip、npm 或者git clone都出各种问题，折腾了好久T_T。只有这个集成的视频下载器是好用的。
谢谢写出这个脚本的Up！！！
新年快乐~！！！

**Comment by LussacZheng** [comment]
感谢你的反馈。

video-downloaader-deploy 本身不会修改系统的环境变量，因此无法在任意终端中调用 you-get, youtube-dl, annie 等命令行工具，包括 Git Bash 。

脚本设计的是以 `下载视频.bat` 作为启动脚本，建议用法是双击打开，或通过其快捷方式双击打开。
它会将以下目录临时地添加进环境变量：

- usr\alias
- usr\command
- usr\python-embed
- usr\python-embed\Scripts
- usr\
- usr\ffmpeg\bin

临时添加，关闭 CMD 窗口后失效，因此支持移动整体文件夹。

当然使用 Git Bash 也可以，只需要手动地、选择性地将以上目录添加进环境变量即可。
